qwik-graphql-tailwind/src/components/issue-tab-view/issue-tab-view.tsx

```diff
@@ -45,22 +51,35 @@ export const IssueTabView = component$(({ activeTab, owner, name }: IssuesProps)
     activeTab: activeTab,
   });
   const dropdownStore = useStore<DropdownStoresProps>({
-    selectedLabel: labelOptions[0].value,
+    selectedLabel: undefined,
```

Comment:
why undefined not null?

Reply:
to be consistent with the milestone value, that needs to be undefined